### PR TITLE
completed test case validate that hte pagination works correctly and test case result 1 page, result 2 page

### PR DESCRIPTION
**Pull request by Micua3105**
validate that the pagination works correcly

### CODE REVIEW

File: src/02-ttts2023-agu.test.js
Comment (liamnguyeniaa) [inline src/02-ttts2023-agu.test.js:1]: this looks like a reformatting code. Please revert the changes and don't include changes that are not related to your code. If you still want to include this change, please create a different branch and make another PR.

File: src/02-ttts2023-agu-pagination-kieuhau.test.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+fixture("ttts2023-agu").page("https://www.agu.edu.vn/vi/tim-kiem");
+
+test("after search, validate that the pagination works correcly", async (t) => {
+  const searchInputSelector = Selector(".search .input-group .search");
+
+  await t.expect(searchInputSelector.exists).ok("Search box should exist");
+
+  await t.typeText(searchInputSelector, "tuyển sinh").pressKey("enter");
+
+  await t
```
Comment (liamnguyeniaa) [inline src/02-ttts2023-agu-pagination-kieuhau.test.js:40]: this is just to validate if the pagination exists. It doesn't validate if the pagination actually works.

**Review by liamnguyeniaa — CHANGES_REQUESTED** [review]
Please fix all the comments and reply as `Done` if it was addressed.

**Comment by liamnguyeniaa** [comment]
Also, the title of the PR should briefly explain what this PR is about. Same as your git commit message.

File: src/02-ttts2023-agu-pagination-kieuhau.test.js (hunk 1 above)
Comment (Micua3105) [reply src/02-ttts2023-agu-pagination-kieuhau.test.js:40]: I will do it again

**Comment by Micua3105** [comment]
Done

File: src/02-ttts2023-agu.test.js
Comment (liamnguyeniaa) [inline src/02-ttts2023-agu.test.js:1]: revert this change.

File: src/02-ttts2023-agu-pagination-kieuhau.test.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { Selector } from "testcafe";
+
+fixture("ttts2023-agu").page("https://www.agu.edu.vn/vi/tim-kiem");
+
+test("validate that the pgination works correctly", async (t) => {
+  const searchInputSelector = Selector(".search .input-group .search");
+  const nextPage = Selector(".main .row .pagination .page-item:last-child .page-link");
```
Comment (liamnguyeniaa) [inline src/02-ttts2023-agu-pagination-kieuhau.test.js]: `nextPageButton`

File: src/02-ttts2023-agu-pagination-kieuhau.test.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  const previousPage = Selector(".main .row .pagination .page-item:first-child .page-link");
```
Comment (liamnguyeniaa) [inline src/02-ttts2023-agu-pagination-kieuhau.test.js]: previousPageButton

File: src/02-ttts2023-agu-pagination-kieuhau.test.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  //click nextPage
+  await t
+    .click(nextPage)
+    .expect(currentPage.innerText).eql("2");
+ 
+  //click previousPage
+  await t
+  .click(previousPage)
+  .expect(currentPage.innerText).eql("1");
+
```
Comment (liamnguyeniaa) [inline src/02-ttts2023-agu-pagination-kieuhau.test.js:61]: validate that the results are changing when switching page.

**Review by liamnguyeniaa — CHANGES_REQUESTED** [review]
comments

File: src/02-ttts2023-agu-pagination-kieuhau.test.js (hunk 2 above)
Comment (Micua3105) [reply src/02-ttts2023-agu-pagination-kieuhau.test.js]: Done

File: src/02-ttts2023-agu-pagination-kieuhau.test.js (hunk 3 above)
Comment (Micua3105) [reply src/02-ttts2023-agu-pagination-kieuhau.test.js]: Done

File: src/02-ttts2023-agu-pagination-kieuhau.test.js (hunk 4 above)
Comment (Micua3105) [reply src/02-ttts2023-agu-pagination-kieuhau.test.js:61]: Done

File: src/02-ttts2023-agu.test.js
Comment (Micua3105) [reply src/02-ttts2023-agu.test.js:1]: Done

File: src/02-ttts2023-agu-pagination-kieuhau.test.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    .expect(currentPageButton.innerText).eql("2");
+
+    const firstPostTitleContent2 = await firstPostTitle.textContent;
+
+  await t  
+    .expect(firstPostTitleContent1).notEql(firstPostTitleContent2);
+
+
+   //click previousPageButton
+  await t
```
Comment (liamnguyeniaa) [inline src/02-ttts2023-agu-pagination-kieuhau.test.js:40]: should we get the content of the post 1 again and ensure that it is the same post since now we are back on page 1?

**Review by liamnguyeniaa — CHANGES_REQUESTED** [review]
looks good so far. Just one more comment.

File: src/02-ttts2023-agu-pagination-kieuhau.test.js (hunk 5 above)
Comment (Micua3105) [reply src/02-ttts2023-agu-pagination-kieuhau.test.js:40]: Done

**Review by liamnguyeniaa — APPROVED** [review]
